.husky/pre-push

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+current_branch=$(git symbolic-ref --short HEAD)
+
+# Prevent push to main branch
+if [ "$current_branch" = "main" ]; then
+  echo "Push to main branch is not allowed"
+  exit 1
+fi
+
+# Run lint:fix and test scripts
+if ! (echo "Running ESLint..." && yarn run lint:fix) ; then
+  echo "Linting failed, push not allowed"
+  exit 1
+fi 
+# if ! yarn test ; then
```

Comment:
I commented this block since I wasn't sure you guys feel it's really useful cause it takes some more time than running eslint. 

Tell me if you think it's okay to have it :)

Reply:
nono, dont want it here, a lot of times we write/fix tests separately while PR is under review